### PR TITLE
[filename] support detect new file status

### PR DESCRIPTION
**Pull request by hiberabyss**
Add `[New]` symbol for new file.

### CODE REVIEW

**Comment by shadmansaleh** [comment]
In what case does unnamed and newfile differ ?

**Comment by hiberabyss** [comment]
> In what case does unnamed and newfile differ ?

For example, when edit a new file via `:e /tmp/newfile.txt`, the status will be `[New]` before `:w`.

**Comment by shadmansaleh** [comment]
Ok can you document this option with other filename options in README.md ?

**Comment by hiberabyss** [comment]
> Ok can you document this option with other filename options in README.md ?

Readme updated, please help to review.

File: README.md [hunk 1]
```diff
@@ -593,6 +593,7 @@ sections = {
     {
       'filename',
       file_status = true,      -- Displays file status (readonly status, modified status)
+      newfile_status = false   -- Whether show new file status, default disabled
```
Comment (shadmansaleh) [inline README.md:596]: I think you should define what you mean by 'new file status'. Also default disabled isn't necessary as =false already states that

Comment (hiberabyss) [reply README.md]: Updated.

**Review by shadmansaleh — APPROVED** [review]
lgtm .

**Pull request merged**